ui/app/src/state/epics/contentTypes.ts

```diff
@@ -53,4 +57,16 @@ const fetchComponentsByContentType: Epic = (action$, state$: Observable<GlobalSt
     )
   );
 
-export default [fetch, fetchComponentsByContentType] as Epic[];
+const dissociateTemplate: Epic = (action$, state$) =>
+  action$.pipe(
+    ofType(dissociateTemplateActionCreator.type),
+    withLatestFrom(state$),
+    switchMap(([{ payload }, { sites: { active } }]) =>
+      dissociateTemplateService(active, payload.contentTypeId).pipe(
+        map(() => dissociateTemplateComplete({ contentTypeId: payload.contentTypeId })), // TODO: send content type id so it can be updated in reducer
+        catchAjaxError(dissociateTemplateFailed)
+      )
+    )
+  );
+
+export default [fetch, fetchComponentsByContentType, dissociateTemplate] as Epic[];
```

Comment:
Please format this file like other epic files: all epics inside the array, each epic with their own `// region {actionName}`, the `as Epic[]` should be `as CrafterCMSEpic[]`